### PR TITLE
feat: payment views core 

### PR DESCRIPTION
**Pull request by gitanjli525**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
core changes for payment ops revamp
small ui change - customize columns button is moved inside the table as shown in the attached screenshot
image

## How did you test it?
code is compiling, ui change as mentioned in description

## Where to test it?

- [x] INTEG
- [x] SANDBOX
- [ ] PROD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/components/LoadedTable.res [hunk 1]
```diff
@@ -829,6 +844,7 @@ let make = (
                 ?showborderColor
                 tableHeadingTextClass
                 nonFrozenTableParentClass
+                showCustomizeColumn={rightTitleElement != React.null ? true : false}
```
Comment (PritishBudhiraja) [inline src/components/LoadedTable.res:847]:
```suggestion
                showCustomizeColumn={rightTitleElement != React.null}
```

File: src/screens/Order/OrderUIUtils.res [hunk 2]
```diff
@@ -427,7 +427,7 @@ let getOrdersList = async (
   try {
     let ordersUrl = getURL(~entityName=ORDERS, ~methodType=Post)
     let res = await updateDetails(ordersUrl, filterValueJson->JSON.Encode.object, Post)
-    let data = res->getDictFromJsonObject->getArrayFromDict("data", [])
+    let data = res->LogicUtils.getDictFromJsonObject->LogicUtils.getArrayFromDict("data", [])
```
Comment (sagarnaikjuspay) [inline src/screens/Order/OrderUIUtils.res:430]: can u open `LogicUtils` at top

File: src/components/LoadedTable.res [hunk 3]
```diff
@@ -906,20 +922,6 @@ let make = (
       </DesktopView>
     </div>
 
-  let customizeColumsButtons = {
-    switch clearFormattedDataButton {
-    | Some(clearFormattedDataButton) =>
-      <div className={`flex flex-row mobile:gap-7 desktop:gap-10 ${filterBottomPadding}`}>
-        clearFormattedDataButton
-        <Portal to={""}> rightTitleElement </Portal>
```
Comment (JeevaRamu0104) [inline src/components/LoadedTable.res:914]: why is this removed

Comment (gitanjli525) [reply src/components/LoadedTable.res:914]: Portal is not required here as {to} is empty string, it is simply rendering the children.

File: src/screens/Order/OrderUIUtils.res (hunk 2 above)
Comment (gitanjli525) [reply src/screens/Order/OrderUIUtils.res]: This change is not a part of this PR, done in next PR.

**Review by JeevaRamu0104 — APPROVED** [review]

**Review by PritishBudhiraja — APPROVED** [review]

**Review by Riddhiagrawal001 — APPROVED** [review]
